bind/gencpy.go

```diff
@@ -243,11 +243,17 @@ func (g *cpyGen) gen() error {
 
 	for _, v := range g.pkg.vars {
 		name := v.Name()
+		get := v.doc
+		set := v.doc
+		if v.doc == "" {
```

Comment:
humm... well, turns out it's not completely straightforward.
`gopy`'s `cpy2` generation code is missing the concept of the step needed by the CPython2 backend to create the necessary C-structures for functions and variables (the top-level ones that are populating the C-module.)

the code you pointed out is actually completely ignoring the `Var.doc` field passed to the `Function` generation code. (because that generation code is only concerned with the wrapping/calling of the C function from CPython. the documentation is only attached during `cpyGen.gen()`.)

For now, I'd be willing to ignore this code duplication.